### PR TITLE
Wishlist page

### PR DESCRIPTION
**Pull request by NeerajVodala**
Added wishlist page
wishlist-page

### CODE REVIEW

File: pages/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (AtulHande03a) [inline pages/wishlist.html:7]: add author tag

**Review by LordUttam — APPROVED** [review]
Looks good to merge. All the best :)

**Review by AtulHande03a — APPROVED** [review]
apart from author meta tag , rest looks good

File: pages/wishlist.html (hunk 1 above)
Comment (NeerajVodala) [reply pages/wishlist.html:7]: Sure, I'll do that after finishing everything.

**Pull request merged**